xapian-applications/omega/templates/query

```diff
@@ -61,6 +62,8 @@ if (!("autofocus" in document.createElement("input")))
 ${Todo: Replace radios with search buttons}
 $if{,$include{inc/anyalldropbox},
 <fieldset>$include{inc/anyallradio}</fieldset>}
+${Search Control(End)}
```

Comment:
I wouldn't bother putting "end" indicators in; there's no nesting so it's obvious when one has finished because the next one has started.